src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -45,15 +38,16 @@ class AvramRoutes {
   // TODO: move/merge this endpoint into a status API
   @ApiMethod(name = "dbPoolStats", httpMethod = "get", path = "dbPoolStats")
   def dbPoolStats: DbPoolStats = {
+    val dataSource = Avram.dbcpDataSource
     val result = for {
-      totalConnections <-  dataSource.database.run(
+      totalConnections <- database.run(
         sql"select count(*) from pg_stat_activity where pid <> pg_backend_pid() and usename = current_user".as[Int])
```

Comment:
Alluded to above, but would be nice to move SQL queries out of the routes, into a DB component layer.